numpy/core/arrayprint.py

```diff
@@ -41,6 +41,7 @@
 from .numerictypes import (longlong, intc, int_, float_, complex_, bool_,
                            flexible)
 from .overrides import array_function_dispatch, set_module
+from operator import index
```

Comment:
```suggestion
import operator
```

Reply:
Is it not beneficial to keep the amount of imported material to a minimum, especially looking at potential performance considerations?

Reply:
That's not how importing works. `from module import x` is just `import module; x = getattr(module, 'x'); del module`